### PR TITLE
add 1.17, remove some jenkins references

### PR DESCRIPTION
**Pull request by MiniDigger**

### CODE REVIEW

File: src/js/downloads.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+        "cache": null,
+    },
     "Paper-1.16": {
         "title": "Paper 1.16.5",
         "api_endpoint": "paper",
         "api_version": "1.16",
-        "jenkins": "Paper-1.16",
         "github": "PaperMC/Paper",
-        "desc": "Active development for the current Minecraft version.",
+        "desc": "Support branch for 1.16.",
```
Comment (MiniDigger) [inline src/js/downloads.js:16]: open for suggestions on this one

**Comment by simpleauthority** [comment]
I had this opened a bit ago https://github.com/PaperMC/papermc.io/pull/89

**Comment by MiniDigger** [comment]
yeah kash linked after I was done, lol

**Comment by simpleauthority** [comment]
no worries, this has more content so either way

**Review by kashike — APPROVED** [review]

**Pull request merged**